oe_theme.theme

```diff
@@ -962,6 +962,9 @@ function oe_theme_preprocess_paragraph__oe_text_feature_media(array &$variables)
   /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
   $paragraph = $variables['paragraph'];
 
+  // Retrieve the correct paragraph translation.
+  $paragraph = \Drupal::service('entity.repository')->getTranslationFromContext($paragraph, $paragraph->language()->getId());
```

Comment:
Why are you passing a language here? It defaults to the current content language but you are passing the original paragraph language so it will do...nothing?

Reply:
You can see now the tests are failing. I noticed during debugging without that it get's reset to the English version so the English get's rendered instead of the Bulgarian.